modules/scheduled_queries/outputs.tf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "query_name" {
```

Comment:
I'm not sure what the purpose of this output is? Concatting all the values does not seem very useful.

Reply:
I took this idea from the pub/sub module. So will have a list of queries in the format `projects/{projectId}/locations/{location}/transferConfigs/{configId}`